src/hostapi/pulseaudio/pa_linux_pulseaudio.c

```diff
@@ -1461,11 +1448,8 @@ PaError PaPulseAudio_RenameSource( PaStream *s, const char *streamName )
     op = pa_stream_set_name( stream->inputStream, streamName, RenameStreamCb, stream );
     PaPulseAudio_UnLock( stream->mainloop );
 
-    /* Wait for completion. */
-    while (pa_operation_get_state( op ) == PA_OPERATION_RUNNING)
-    {
-        pa_threaded_mainloop_wait( stream->mainloop );
-    }
+    PaPulseAudio_ReleaseOperation( stream->hostapi,
+                                   &op );
```

Comment:
The code that this replaces did not call pa_operation_unref().
Does this new code fix an old problem with missing unref calls?

Reply:
`pa_operation_unref` should be called after process is completed. Code was incorrect.